mkdocs/contrib/search/search_index.py

```diff
@@ -127,34 +128,6 @@ def generate_search_index(self):
 
         return data
 
-    def strip_tags(self, html):
-        """strip html tags from data"""
-        s = HTMLStripper()
-        s.feed(html)
-        return s.get_data()
-
-
-class HTMLStripper(HTMLParser):
-    """
-    A simple HTML parser that stores all of the data within tags
-    but ignores the tags themselves and thus strips them from the
-    content.
-    """
-
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-        self.data = []
-
-    def handle_data(self, d):
-        """
-        Called for the text contents of each tag.
-        """
-        self.data.append(d)
-
-    def get_data(self):
-        return '\n'.join(self.data)
```

Comment:
Right, I missed the fact that it used to be joined with `'\n'`, just assumed it's `''` somehow.